test/wordsearch.js

```diff
@@ -2,6 +2,9 @@
 
 const assert = require('assert');
 const {WordSearchSolver, WordSearchDirection} = require('../');
+const {
+  WordSearchSpaceTreatment,
+} = require('../build/src/WordSearch/WordSearchSolver');
```

Comment:
We only want to import from the index. You should be able to export multiple items from the one module.

Reply:
I went ahead and verified that what I was suggesting would work:

```diff
diff --git a/src/index.ts b/src/index.ts
index 2e174ff..7d25a9e 100644
--- a/src/index.ts
+++ b/src/index.ts
@@ -32,4 +32,7 @@ export {SemaphoreStream} from './Semaphore/SemaphoreStream';
 export {Point as WordSearchPoint} from './WordSearch/Point';
 export {Result as WordSearchResult} from './WordSearch/Result';
 export {WordSearchDirection} from './WordSearch/WordSearchDirection';
-export {WordSearchSolver} from './WordSearch/WordSearchSolver';
+export {
+  WordSearchSolver,
+  WordSearchSpaceTreatment,
+} from './WordSearch/WordSearchSolver';
diff --git a/test/wordsearch.js b/test/wordsearch.js
index 76487eb..a9f084d 100644
--- a/test/wordsearch.js
+++ b/test/wordsearch.js
@@ -1,10 +1,11 @@
 /* global describe, it */
 
 const assert = require('assert');
-const {WordSearchSolver, WordSearchDirection} = require('../');
 const {
+  WordSearchDirection,
+  WordSearchSolver,
   WordSearchSpaceTreatment,
-} = require('../build/src/WordSearch/WordSearchSolver');
+} = require('../');
 
 function assertResultsContainsWord(results, word) {
   for (const result of results) {
```

Reply:
Thanks :)

Sorry I kinda forgot about this 😇